### PR TITLE
Address sphinx build warnings

### PR DESCRIPTION
**Pull request by katrogan**

### CODE REVIEW

**Review by wild-endeavor — APPROVED** [review]

File: rsts/tutorials/run_flyte_hosted.rst [hunk 1]
```diff
@@ -4,7 +4,7 @@
 [Optional] Run Your Workflow on a Hosted Flyte
```
Comment (kumare3) [inline rsts/tutorials/run_flyte_hosted.rst:4]: I thought you were going to move the contents of this file to a tab in the previous page?

Comment (katrogan) [reply rsts/tutorials/run_flyte_hosted.rst:4]: @kumare3 this PR is just warnings, but I can do it in the same one if you prefer?

Comment (katrogan) [reply rsts/tutorials/run_flyte_hosted.rst]: done

**Review by wild-endeavor — APPROVED** [review]

File: rsts/tutorials/run_flyte_laptop.rst [hunk 2]
```diff
@@ -68,6 +68,16 @@ Steps
 
     .. tab:: Using Docker for Mac
 
+        Coming soon
+
+    .. tab:: Hosted Flyte
+
+        Refer to the :ref:`howto-sandbox-dedicated-k8s-cluster` guide.
+
+        Once you've deployed flyte sandbox to a cloud provider, refer to :ref:`tutorials-run-flyte-laptop` to run your first workflow remotely.
```
Comment (kumare3) [inline rsts/tutorials/run_flyte_laptop.rst:77]: circular reference? What you want to say is you can jump to the running section below?

Comment (katrogan) [reply rsts/tutorials/run_flyte_laptop.rst:77]: it links to that exact section below

Comment (katrogan) [reply rsts/tutorials/run_flyte_laptop.rst]: updated

Comment (katrogan) [reply rsts/tutorials/run_flyte_laptop.rst]: @kumare3

**Review by wild-endeavor — APPROVED** [review]

**Pull request merged**